packaging/aznfs/RPM/aznfs.spec

```diff
@@ -32,6 +36,59 @@ if [ "$init" != "systemd" ]; then
 	exit 1
 fi
 
+if grep -i "mariner" /etc/os-release > /dev/null 2>&1; then
+	# Check if stunnel is not already installed.
+	if ! command -v stunnel > /dev/null 2>&1; then
+		# Install stunnel from source on Mariner.
+		wget https://www.stunnel.org/downloads/stunnel-latest.tar.gz -P /tmp
+		if [ $? -ne 0 ]; then
+			echo "Failed to download stunnel source code. Please install stunnel and try again."
+			exit 1
+		fi
+
+		tar -xvf /tmp/stunnel-latest.tar.gz -C /tmp
+		if [ $? -ne 0 ]; then
+			echo "Failed to extract stunnel tarball. Please install stunnel and try again."
+			cd -
+			rm -f /tmp/stunnel-latest.tar.gz
+			exit 1
+		fi
+
+		stunnel_dir=$(tar -tf /tmp/stunnel-latest.tar.gz | head -n 1 | cut -f1 -d'/')
+
+		cd /tmp/$stunnel_dir
+		./configure
```

Comment:
pls make sure this works on the standard mariner and doesn't complain about some missing package.

Reply:
These changes are tested on Mariner linux. The test release containing these changes is 0.1.387.